recipes/genform/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "r8" %}
+{% set name = "genform" %}
+{% set sha256 = "d9199aa9c13c6aa68025b15e4d30a4e445a265ce828b3314103307dc7b6c13b5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+#  skip: True  # [osx]
+
+source:
+  # download is a stable link for https://sourceforge.net/code-snapshots/svn/g/ge/genform/code/genform-code-r8-trunk.zip
+  url: https://depot.galaxyproject.org/software/genform/genform_r8_src_all.zip
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+
+# genform without parameter or w help has exit code 1, so we need test data
+# since test data can't be taken from the source dir its created manually
+test:
+  commands:
+    - 'echo -e "132.07686\t147049184\n133.08012\t4004960.8" > CreatineMs.txt && genform ms=CreatineMs.txt'
+about:
+  home: https://sourceforge.net/projects/genform/
+  license: 'LGPL (>= 2.0)'
```

Comment:
I checked if the `>= 2.0` is accurate. Acc. to https://sourceforge.net/projects/genform/ and the `licence_file`, this should rather be `LGPL-2.0-only`.

Acc. to the source files: Well... 
```
> grep -rhim1 license genform-code-r8-trunk/ | sort -u                                                                                                                                                                                                                           
                  GNU LIBRARY GENERAL PUBLIC LICENSE
 *  it under the terms of the GNU General Public License as published by
 *  it under the terms of the GNU General Public License as published by
```
... Y U NO GIVE CORRECT LICENSE? \*sigh\*

So... even though the SourceForge project page and `src/lgpl-2.0.txt` say "LGPL", it is actually "GPL" with an unknown version. Großartig!

Reply:
I think its OK. The file actually says: 

```
                  GNU LIBRARY GENERAL PUBLIC LICENSE
                       Version 2, June 1991
```

which is LGPL2 according to https://en.wikipedia.org/wiki/GNU_Lesser_General_Public_License

Reply:
Yes, that's what the license file says. However, this is just the generic license text. It is more important how the thing is actually licensed, i.e., the file headers which all say something like
```
>  head -n12 genform-code-r8-trunk/src/main.cpp
/*
 *  main.cpp, part of GenForm by M. Meringer Copyright (C) 2015
 *
 *  This program is free software; you can redistribute it and/or modify
 *  it under the terms of the GNU General Public License as published by
 *  the Free Software Foundation.
 *
 *  This program is distributed in the hope that it will be useful,
 *  but WITHOUT ANY WARRANTY; without even the implied warranty of
 *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 *  GNU General Public License for more details.
*/
```

Reply:
Damn - you are right. I could contact the maintainer to change the text and/or the license file. But then the new release would need to be in cargo port before we can have it here. Which needs time -- but "gut ding will weile haben"

Do you see another option?

Reply:
> I could contact the maintainer to change the text and/or the license file.

That would be most helpful, yes, please do! Thanks!

> Do you see another option?

I'm not an expert on the \*GPLs as I never released something under such a license and thus avoided dissecting those (comparably) long license texts. But according to our "scientifically sound source" \*cough\* https://en.wikipedia.org/w/index.php?oldid=901931174#Compatibility :
> One feature of the LGPL is the permission to relicense under the GPL any piece of software which is received under the LGPL

Which would mean we are free to distribute it under the GPL even if it would be distributed upstream as LGPL.

And since this package is just a single executable and not a library anyway, the LGPL / GPL distinction doesn't matter much.

Ergo: I'm fine to tag it as `GPL`. And since `LGPL 2.0` has been "mentioned" upstream, it would probably be okay to say `GPL-2.0-only`, I guess.